lib/thunderbird_common.pm

```diff
@@ -136,14 +135,15 @@ sub tb_send_message {
             release_key "alt";
             send_key "tab";
             send_key "ret";
-            assert_and_click "thunderbird_confirm_security_exception";
             # Now, return focus to thunderbirt sent email window.
             hold_key "alt";
             send_key "f1";
             release_key "alt";
             send_key "tab";
             send_key "tab";
             send_key "ret";
+            assert_and_click "thunderbird_confirm_security_exception";
+            #assert_and_click "pare_aqui";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
OMG, that comment completely escaped :(